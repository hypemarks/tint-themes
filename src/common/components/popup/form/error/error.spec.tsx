import { render } from '@testing-library/react';
import React from 'react';
import Error from './error';

describe('Test <Error /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(<Error show={true} message={'ERROR'} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
