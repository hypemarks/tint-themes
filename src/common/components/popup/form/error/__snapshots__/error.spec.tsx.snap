// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Error /> Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="error undefined"
  >
    ERROR
  </div>
</DocumentFragment>
`;
